src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtCasesToNomisIntTest.kt

```diff
@@ -117,6 +127,38 @@ class CourtCasesToNomisIntTest : SqsIntegrationTestBase() {
                     NOMIS_NEXT_COURT_APPEARANCE_ID.toString(),
                   ),
                 ),
+              )
+              .withRequestBody(
+                WireMock.matchingJsonPath(
```

Comment:
very, very minor - but worth doing a static import of `Wiremock.equalTo` to reduce the noice in this code

Reply:
yes, must have been missed